crypto/bls/bls_crypto.py

```diff
@@ -14,8 +16,19 @@ def load_group_params(self) -> GroupParams:
 
 class BlsCryptoSigner(metaclass=ABCMeta):
     def __init__(self, sk: str, pk: str, params: GroupParams):
-        assert sk
-        assert pk
+
```

Comment:
This is rather Design by contract. Asserts or exceptions here?

Reply:
as of now `sk` and `pk` come from files, thus, exceptions seems as better choice here